src/sass/app_2/_common/components/settings_dialog.scss

```diff
@@ -0,0 +1,218 @@
+                    border-bottom: 1px solid $COLOR_GRAY;
+                }
+                &:hover {
+                    background-color: $COLOR_LIGHT_GRAY;
+                }
+            }
+        }
+
+    }
+    .language-dialog-container {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        pointer-events: none;
+        background: $COLOR_LIGHT_GRAY;
+        border-radius: $BORDER_RADIUS;
+        opacity: 0;
+        transition: all 0.3s ease 0.15s;
+        overflow: hidden;
+        overflow-y: scroll;
+
+        &.show {
+            pointer-events: auto;
+            position: relative;
+            opacity: 1;
+            height: 280px;
+        }
+        .language-header {
+            font-size: 12px;
+            padding: 8px 16px;
+            display: flex;
+            background: $COLOR_WHITE;
+            border-bottom: 1px solid $COLOR_GRAY;
+            cursor: pointer;
+            text-decoration: none;
+
+            svg.arrow-back {
+                padding-right: 4px;
+            }
+        }
+        .language-container {
+            display: grid;
+            padding: 8px 16px 0px;
+
+            .language-row {
+                display: flex;
+                justify-content: flex-start;
+                align-items: center;
+                background: $COLOR_WHITE;
+                height: 32px;
+                cursor: pointer;
+
+                &:hover {
+                    background: $COLOR_LIGHT_GRAY;
+                }
+                &:nth-child(odd) {
+                    grid-column: 1;
+                    border: 1px solid $COLOR_GRAY;
+                    border-bottom-width: 0;
+                }
+                &:nth-child(even) {
+                    grid-column: 2;
+                    border: 1px solid $COLOR_GRAY;
+                    border-left-width: 0;
+                    border-bottom-width: 0;
+                }
+                &:last-child:not(:nth-child(even)) {
+                    border-top-width: 1px;
+                }
+                &:nth-last-child(-n+2) {
+                    border-bottom-width: 1px;
+                }
+                i.flag {
+                    padding-left: 8px;
+                    padding-right: unset;
+                }
+                span {
+                    padding-left: 8px;
+                    font-size: 12px;
+                }
+                a {
+                    width: 100%;
+                    height: 100%;
+                    display: flex;
+                    align-items: center;
+                }
+            }
+        }
+    }
+    & > .language-dialog-container {
+        pointer-events: none;
+    }
+    &.show > .language-dialog-container.show {
+        pointer-events: auto;
+    }
+    i.flag {
+        padding-right: 6px;
+
+        &.ic-flag-en:before {
+            content: url('../images/trading_app/footer/language/ic_flag_en.svg');
```

Comment:
`images/app_2/...` as `images/trading_app` does not exist anymore.

Also might be worth considering different folder for the flags. Since they're more general, better to not assign them to the location _currently_ used in design (`footer`). 🙂

Reply:
I'm planning to refactor all of our icons in preparation for the dark theme card. I'll handle the restructuring of the icons and convert some to inline svg in a different card. :)